mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Requests tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+
+  Scenario: Given an item Id, a user Id, and a pickup location, attempt to create a page request when the applicable request policy disallows pages
+    * def itemId = call uuid1
+    * def servicePointId = call uuid1
+    * def userId = call uuid1
+    * def userBarcode = 'FAT-1030UBC'
+    * def itemBarcode = 'FAT-1030IBC'
+    * def groupId = call uuid1
+    * def instanceId = call uuid1
+    * def locationId = call uuid1
+    * def holdingId = call uuid1
+    * def materialTypeId = call uuid1
+    * def materialTypeName = 'book'
+    * def extRequestTypes = ["Hold", "Recall"]
+
+    # post a material type and policies
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(materialTypeId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPolicies') { extRequestTypes: #(extRequestTypes) }
```

Comment:
there are 5 different policies in that method and it isn't really clear for which of them variable "extRequestTypes" are being sent. So I guess let's rename it into extRequestTypesFor...Policy

Reply:
done